app/policies/file_entry_policy.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class FileEntryPolicy < TeamDependantPolicy
+  def index?
+    true
```

Comment:
make sure this is correct. 
permission is depending on account / team membership ?